components/MintHypercerts.tsx

```diff
@@ -26,7 +26,7 @@ export function MintHypercerts({ data }: MintHypercertProps) {
     return encodeFunctionData({
       abi: HypercertMinterAbi,
       functionName: "mintClaim",
-      args: [address, parseEther("1"), metadata, 0],
+      args: [address, 10000, metadata, 0],
```

Comment:
It should be 1 ether

Reply:
Prior 17 claims has minted with this value